src/Web/Action/LoginAction.php

```diff
@@ -28,7 +31,7 @@ public function __invoke(Request $request, Response $response, $args)
     {
         $data = $request->getParsedBody();
         $userEmail = filter_var($data['email'], FILTER_SANITIZE_STRING);
-        $userPass = filter_var($data['password'], FILTER_SANITIZE_STRING);
+        $userPass = md5(filter_var($data['password'], FILTER_SANITIZE_STRING));
```

Comment:
`md5` maybe for a demo app, but not a proper way to handle password authentication in 2016.